src/parser/parser_utils.js

```diff
@@ -187,6 +187,32 @@ function splitVAST(ads) {
   return splittedVAST;
 }
 
+/**
+ * Parses the attributes and assign them to object
+ * @param  {Object} attributes attribute
+ * @param  {Object} verificationObject with properties which can be assigned
+ */
+function assignAttributes(attributes, verificationObject) {
+  if (attributes) {
+    for (const attrKey in attributes) {
+      const attribute = attributes[attrKey];
+
+      if (
+          attribute.nodeName &&
+          attribute.nodeValue &&
+          verificationObject.hasOwnProperty(attribute.nodeName)
+      ) {
+        let value = attribute.nodeValue;
+
+        if (typeof verificationObject[attribute.nodeName] === 'boolean') {
+          value = parseBoolean(value);
+        }
+        verificationObject[attribute.nodeName] = value;
```

Comment:
instead of mutating verificationObject, you could clone it, make the modifications and return the new object